Public/Src/Utilities/Configuration/ISandboxConfiguration.cs

```diff
@@ -243,5 +245,15 @@ public interface ISandboxConfiguration
         /// Thus, this enforcement is made opt-in.
         /// </remarks>
         bool EnsureTempDirectoriesExistenceBeforePipExecution { get; }
+
+        /// <summary> 
+        /// Paths which should be untracked for all processes 
+        /// </summary> 
+        IReadOnlyList<AbsolutePath> GlobalUnsafeUntrackedScopes { get; }

```

Comment:
>Unsafe [](start = 42, length = 6)

For both new configurations, please add some information why they are unsafe.

Reply:
Please add this before you check in.

---
In reply to: [292589107](https://github.com/microsoft/BuildXL/pull/442#discussion_r292589107) [](ancestors = 292589107)

Reply:
added